src/components/logs/app/main.js

```diff
@@ -26,7 +26,7 @@ window.addEventListener('message', (event) => {
             const { containers, colors } = message;
             schemaColors = JSON.parse(colors);
             if (containers.length === 1) {
-                defaultContainer = containers[0];
+                defaultContainer = containers[0].name;
```

Comment:
💡 observation only, I think the `containers[0].name` is safe operation rather then checking: `if value-exist case` hence in that spirit this looks safe, please let me know if that is not the case. Looks good to me and I can cut release for this once this is merged.

Thank you. 🙏

Reply:
I didn't get what you're trying to say 😅 you're asking if we should check if the name property exists? As i filter empty names from https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/pull/1188/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R1398 i guess that check would be superfluous.

Reply:
Thank you! 🙂☕️ Sounds good, basically I was trying to archive the intention more obvious for anyone else see this in future. Sounds good. Feel free to merge it and I can cut a release again, if that sounds good.